service-front/config/autoload/local.php.dist

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Again, since we'll generally configure any dev-level config in docker-compose this can be removed.